tasks.py

```diff
@@ -0,0 +1,426 @@
+from invoke import task, run 
```

Comment:
Why isn't invoke listed under the dev dependencies header of the Pipenv file? Strange that this was even able to run!

Reply:
I had already installed it through pip. That's why I had it. Anyways, it should be added when I run these commands in the env from pipenv. 

This brings the question of how these tasks will be used. I was thinking of this order. 
1. Pipenv shell 
2. Pipenv install invoke 
3. invoke setup 

.... after looking at this, I am wondering why I am installing pipenv as part of one of the tasks. I think I should go ahead and remove pipenv from packages to be installed.